app/flask_config.py

```diff
@@ -5,11 +5,7 @@
 class Config:
     """Base configuration variables."""
 
-    TRELLO_API_TOKEN = os.environ.get("TRELLO_API_TOKEN")
-    TRELLO_API_KEY = os.environ.get("TRELLO_API_KEY")
-    TRELLO_BOARD_ID = os.environ.get("TRELLO_BOARD_ID")
-
-    if not (TRELLO_API_KEY and TRELLO_API_TOKEN and TRELLO_BOARD_ID):
-        raise ValueError(
-            "Missing config for Flask application. Did you forget to run setup.sh?"
-        )
+    MONGO_URI = os.environ.get("MONGO_URI")
```

Comment:
I assume your comment about being unable to run the tests directly you mean `poetry run pytest` doesn't pass. This is why you are not able to run all the tests together - this line setting MONGO_URI is only evaluated once ever (the first time your app is created)

You can still successfully run `poetry run pytest tests`/`poetry run pytest tests_e2e` separately.
If you want to run them all together, you will need to change this to an instance attribute and change the line in `app/__init__.py` to something like `app.config.from_object(Config())`